src/sentry/static/sentry/app/views/settings/account/accountSecurity/accountSecurityEnroll.jsx

```diff
@@ -165,8 +165,8 @@ class AccountSecurityEnroll extends AsyncView {
   loadOrganizationContext = () => {
     if (this.invite && this.invite.memberId) {
       fetchOrganizationByMember(this.invite.memberId, {
-        loadOrg: true,
-        setActive: true,
+        addOrg: true,
```

Comment:
Is this the only place we call `fetchOrganizationByMember`?

Reply:
Yes